content/participants/ttjc@shvmsrma/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Shivam Sharma
+type: "participant"
+name: "Shivam Sharma"
+portfolio: "https://shvmsrma.netlify.com"
+github-link: "https://github.com/shvmsrma"
+linkedin-link: "https://www.linkedin.com/in/shivam-sharma-2a312612b/"
+looking-for: "job"
+description: "Learning React and Node"
+---
+
+Hi! I am a Shivam Sharma let me walk you through my intro, projects and blogs.
+
+### introduction
+
+I am learning Javascript 
+
+## why are you taking this challenge?
+
+I need an Job in june 2020.
+I want to learn.
+
+## something interesting about you?
+I am a state level badminton player
+
+### help I need?
+
+It would be good if someone tells me about graphql.
+
+### how to contact me?
+
+-[Twitter](https://twitter.com/Shvmsrma)
+-[GitHub](https://github.com/shvmsrma)
```

Comment:
add space after - 
use proper markdown

Reply:
Sorry for the mistake. I have resolved it.